atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/keyvalue/api/cache/LockWatchValueScopingCacheImplTest.java

```diff
@@ -311,14 +306,6 @@ public void leaderElectionCausesCacheToBeCleared() {
         // Confirms entry is no longer present
         TransactionScopedCache scopedCache3 = valueCache.getTransactionScopedCache(TIMESTAMP_3);
         assertThat(getRemotelyReadCells(scopedCache3, TABLE, CELL_1)).containsExactlyInAnyOrder(CELL_1);
-        eventCache.processGetCommitTimestampsUpdate(
-                ImmutableList.of(TransactionUpdate.builder()
-                        .startTs(TIMESTAMP_3)
-                        .commitTs(999_999_999)
-                        .writesToken(LockToken.of(UUID.randomUUID()))
-                        .build()),
-                LockWatchStateUpdate.success(newLeader, -1L, ImmutableList.of()));
-        valueCache.updateCacheOnCommit(ImmutableSet.of(TIMESTAMP_3));
```

Comment:
Not entirely sure what this code was aiming to do

Reply:
encouraging 🔪